src/wp-includes/class-wp-user-query.php

```diff
@@ -1043,10 +1045,11 @@ protected function parse_orderby( $orderby ) {
 	 * Generate cache key.
 	 *
 	 * @since 6.3.0
+	 * @since 6.9.0 $args is no longer used.
```

Comment:
This seems the most common approach elsewhere in core.

```suggestion
	 * @since 6.9.0 The `$args` parameter was deprecated and renamed to `$deprecated`.
```